targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_MIMXRT1050/drivers/fsl_lpspi.c

```diff
@@ -258,9 +258,9 @@ void LPSPI_MasterGetDefaultConfig(lpspi_master_config_t *masterConfig)
     masterConfig->cpha         = kLPSPI_ClockPhaseFirstEdge;
```

Comment:
@artokin Can you revert the change to file attributes (from 10644 to 10755), only this file

Reply:
@0xc0170 , made in the next commit.